buildenv/jenkins/JenkinsfileBase

```diff
@@ -193,6 +193,37 @@ def setup() {
 			OPENJ9_BRANCH_OPTION = (params.OPENJ9_BRANCH) ? "--openj9_branch ${params.OPENJ9_BRANCH}" : ""
 			OPENJ9_SHA_OPTION = (params.OPENJ9_SHA) ? "--openj9_sha ${params.OPENJ9_SHA}" : ""
 
+			// system test repository exports to be used by systemtest/common.xml
+			if (params.ADOPTOPENJDK_SYSTEMTEST_REPO) {
+				env.OPENJDK_SYSTEMTEST_REPO = "${params.ADOPTOPENJDK_SYSTEMTEST_REPO}" 
+			}
+			
+			if (params.ADOPTOPENJDK_SYSTEMTEST_BRANCH) {
+				env.OPENJDK_SYSTEMTEST_BRANCH = "${params.ADOPTOPENJDK_SYSTEMTEST_BRANCH}"
+			} else {
+				echo "ADOPTOPENJDK_SYSTEMTEST_BRANCH not set"
```

Comment:
This is an optional parameter and it is only used for system test. I would prefer not to echo it in the base. If msg is needed, please do it in systemtest common.xml. Same for the rest echo msg.

Reply:
Agreed, removed those else{..} blocks.